src/qt/bitcoingui.cpp

```diff
@@ -370,13 +370,12 @@ void BitcoinGUI::createActions()
         connect(openAction, &QAction::triggered, this, &BitcoinGUI::openClicked);
         connect(m_open_wallet_action->menu(), &QMenu::aboutToShow, [this] {
             m_open_wallet_action->menu()->clear();
-            std::vector<std::string> available_wallets = m_wallet_controller->getWalletsAvailableToOpen();
-            std::vector<std::string> wallets = m_node.listWalletDir();
-            for (const auto& path : wallets) {
+            for (auto i : m_wallet_controller->listWalletDir()) {
```

Comment:
I think you still want to capture by reference here, `auto&` or `const std::pair<const std::string, bool>&`.

Reply:
Done.